ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/common/LedgerIdMode.scala

```diff
@@ -3,19 +3,36 @@
 
 package com.digitalasset.platform.common
 
+import com.digitalasset.daml.lf.data.Ref
 import com.digitalasset.ledger.api.domain.LedgerId
 
-sealed abstract class LedgerIdMode extends Product with Serializable
+sealed abstract class LedgerIdMode extends Product with Serializable {
+  def or(other: => LedgerId): LedgerId
```

Comment:
This is effectively a noop when called over `Static`. :thinking: 

Stupid question: do we really need this type? Or can we do with just `Option[LedgerId]`?

Not necessarily an issue for this PR, so don't feel compelled to address this here.

Reply:
I got the feeling it's for nicer Java compatibility, but I have no real evidence.